src/kv.cpp

```diff
@@ -0,0 +1,28 @@
+#include <unordered_map>
+
+#include "common.hpp"
+#include "kv.hpp"
+
+using Hashmap = std::unordered_map<Key, Value>;
+
+template <>
+KV<Hashmap>::KV() noexcept {
+  store = Hashmap();
+}
+
+template <>
+std::optional<Value> KV<Hashmap>::get(const Key &key) noexcept {
+  auto x = store.find(key);
+  if (x == store.end()) return None;
+  return Some(x->second);
+}
+
+template <>
+void KV<Hashmap>::put(const Key &key, Value &value) noexcept {
+  store.emplace(key, value);
+}
+
+template <>
+void KV<Hashmap>::del(const Key &key) noexcept {
+  store.erase(key);
+}
```

Comment:
Apparently, this is a naive implementation of our core interface - the `KV` class. We are supposed to implement it with a practical data structure like Skiplist.

Actually, why don't we follow the pattern of introducing a naive implementation and gradually replacing that with a real one?